apps/alerting/notifications/pkg/apis/alerting_manifest.go

```diff
@@ -6,9 +6,12 @@
 package apis
 
 import (
-	"encoding/json"
+	"fmt"
 
 	"github.com/grafana/grafana-app-sdk/app"
+	"github.com/grafana/grafana-app-sdk/resource"
+
+	v0alpha1 "github.com/grafana/grafana/apps/alerting/notifications/pkg/apis/alerting/v0alpha1"
```

Comment:
@IfSentient -- fyi, codegen creates:
```
v0alpha1 "github.com/grafana/grafana/apps/alerting/notifications/pkg/apis/notifications/v0alpha1"
```

I'm not really sure what are desired structure is 🤷🏻  but i manually changed this since it seems like the smaller change